Source/EasyNetQ/Producer/Rpc.cs

```diff
@@ -328,11 +328,11 @@ protected virtual void RequestPublish<TRequest>(TRequest request, string routing
             advancedBus.Publish(exchange, requestMessage.Properties.ReplyTo, false, responseMessage);
         }
 
-        protected virtual void OnResponderFailure<TRequest, TResponse>(IMessage<TRequest> requestMessage, string exceptionMessage, Exception exception)
+        protected virtual void OnResponderFailure<TRequest, TResponse>(IMessage<TRequest> requestMessage, string exceptionMessage, Exception exception, Type responseType)
             where TRequest : class
             where TResponse : class
         {
-            var responseMessage = new Message<TResponse>();
+            var responseMessage = new Message<TResponse>((TResponse)Activator.CreateInstance(responseType));
```

Comment:
No need in  additional argument, `Activator.CreateInstance(typeof(TResponse))` will be non breaking change.

Reply:
The unnecessary argument is removed.